envs/monkey_zoo/configs/fullTest.conf

```diff
@@ -62,7 +62,7 @@
       "exploiter_classes": [
         "SmbExploiter",
         "WmiExploiter",
-        "SSHExploiter",
+        "RdpExploiter",
```

Comment:
Does it work? 🤔

Reply:
We don't have a vulnerable machine for it, but I added it for scanning purposes.